geode-assembly/geode-assembly-test/src/main/java/org/apache/geode/rest/internal/web/controllers/PutKeyFunction.java

```diff
@@ -30,6 +30,7 @@
 
   private static final String ID = "PutKeyFunction";
 
+  @SuppressWarnings("unchecked")
   @Override
```

Comment:
Is it possible to add correct generic types here so we don't need to suppress anything?

Reply:
I have removed this change. I don't know how it snuck in. It caused a bunch of merge conflicts. The build is fine without this change. If someone wants to clean this test code up I think it should be on another PR.